src/zrc/conf/base.py

```diff
@@ -380,3 +380,6 @@
 IS_HTTPS = os.getenv('IS_HTTPS', '1').lower() in ['true', '1', 'yes']
 
 NOTIFICATIONS_KANAAL = 'zaken'
+
+
+REST_FRAMEWORK['DEFAULT_AUTHENTICATION_CLASSES'] = ()
```

Comment:
@sergei-maertens ik weet niet of dit de goede manier is om dit op te lossen, blijkbaar heeft DRF standaard `BasicAuthentication` en `SessionAuthentication` als authentication classes, maar worden die wel gebruikt door de componenten?

Reply:
Deze moeten er inderdaad niet in zitten.

Kan je dit weer verwijderen en toevoegen aan vng-api-common?
Je zal dan override_settings, waar je deze auth classes leeg maakt, voor de eerste test moeten gebruiken om dit gedrag te simuleren.